### PR TITLE
feat: navigation added

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change Logs

feat: navigation added

## Screenshot
image

## Preview Link
https://deploy-preview-16--ripple-ui.netlify.app/components/navigation/navigation

### CODE REVIEW

File: components/navigation/src/html/standard-navigation.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<div class="card basic-card card-shadow">
+    <ul class="rui-drawer-content--list no-list">
```
Comment (bharati-21) [inline components/navigation/src/html/standard-navigation.html:2]: The accepted convention is that a class name shouldn't be more than 3 characters (incl. modifier).

I suggest choosing a much more precise name.

**Review by bharati-21 — APPROVED** [review]
Looks good to me!

File: components/navigation/src/html/standard-navigation.html (hunk 1 above)
Comment (sonishreyas) [reply components/navigation/src/html/standard-navigation.html:2]: ok will make that change after completing assignment

**Review by rishmi5h — APPROVED** [review]
looks good.

**Pull request merged**